lib/app/color.ex

```diff
@@ -0,0 +1,90 @@
+defmodule App.Color do
+  @hex_colors %{
+    "red-300" => "#FCA5A5",
```

Comment:
Thank you for extracting these hex values. 👌 
I definitely find the **`hue-saturation`** e.g. **`blue-400`** to be more intuitive than **`#60A5FA`** ... 💭